python/llm/portable-zip/README.md

```diff
@@ -0,0 +1,37 @@
+# BigDL-LLM Portable Zip For Windows: User Guide
+
+## Introduction
+
+This portable zip includes everything you need to run LLM (except models). Please refer to How to use section to get started.
```

Comment:
We could add a jump link here for How to use

sth like
```
[How to use](#how-to-use)
```

Reply:
`This portable zip includes everything you need to run an LLM with BigDL-LLM optimizations (except models) . Please refer to [How to use](#how-to-use) section to get started.`

Reply:
Already done as you said : )